scaleway/resource_instance_volume.go

```diff
@@ -34,7 +34,7 @@ func resourceScalewayInstanceVolume() *schema.Resource {
 				ForceNew:    true,
```

Comment:
We need to add `Optional:    true` and `Default: instance.VolumeTypeBSSD.String()`. Cf. [here](https://github.com/terraform-providers/terraform-provider-scaleway/pull/216/files#diff-136bae47c996c61549b8078d867e3cacR33)